src/client/systems/syscommand.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Conquer Space
+*/
+#pragma once
+
+#include "client/systems/sysgui.h"
+
+namespace conquerspace {
+namespace client {
+namespace systems {
+class SysCommand : public SysUserInterface {
+ public:
+    explicit SysCommand(conquerspace::engine::Application& app)
+        : SysUserInterface(app) {}
+
+    void Init();
+    void DoUI(int delta_time);
+    void DoUpdate(int delta_time);
```

Comment:
You might want to override `Interval` to update ships at a faster interval.
`int Interval() { return 1; }` would enable ships to be updated every tick.